e2etest/pkg/k8s/pods.go

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package k8s
+
+import (
+	"bytes"
+	"context"
+	"io"
+
+	corev1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+func GetPodLogs(cs clientset.Interface, pod *corev1.Pod, podLogOpts corev1.PodLogOptions) (string, error) {
```

Comment:
Just "PodLogs", also I'd just have the containerName as parameter, and hardcode the # of lines

Reply:
fixed